docs/tutorials/introduction.md

```diff
@@ -125,20 +125,23 @@ async def ping_local():
 The await keyword must be used within another function (typically an asyncio function). Otherwise, it will result in a syntax error.
 
 ## Matchers available
-Matchers are used to match a message, sent by a user, to a connector and a skill. Opsdroid comes ready with 8 different matchers, each one of them has its own settings and specification.
+Matchers are used to match a message, sent by a user, to a connector and a skill. Opsdroid comes ready with 9 different matchers, each one of them has its own settings and specification.
 
 - Basic matcher
-  - [Regular Expressions](/matchers/regex.md)
-- NPL Matchers
-  - [Dialogflow(previous Api.ai)](/matchers/dialogflow.md)
-  - [Wit.ai](/matchers/wit.ai.md)
-  - [Lui.ai](/matchers/luis.ai.md)
+  - [Regular Expressions](/docs/matchers/regex.md)
+- NLP Matchers
+  - [Dialogflow(previous Api.ai)](/docs/matchers/dialogflow.md)
+  - [Wit.ai](/docs/matchers/wit.ai.md)
+  - [Luis.ai](/docs/matchers/luis.ai.md)
+  - [Recast.ai](/docs/matchers/recast.ai.md)
+- NLU Matcher
+  - [Rasa_NLU](/docs/matchers/rasanlu.md)
 - Special Matcher
-  - [Always](/matchers/always.md)
-  - [Crontab](/matchers/crontab.md)
-  - [Webhook](/matchers/webhook.md)
+  - [Always](/docs/matchers/always.md)
+  - [Crontab](/docs/matchers/crontab.md)
+  - [Webhook](/docs/matchers/webhook.md)
 
-Read the [Matchers overview page](matchers/overview) for a quick reference guide on how to use them.
+Read the any of the matchers page for a quick reference guide on how to use them.
```

Comment:
I'm happy with this change, but let's remove `the` before your change making the sentence:
"Read any of the matchers page for a quick (...)."